server/mentionscrawler/models/user.py

```diff
@@ -0,0 +1,22 @@
+from flask_sqlalchemy import SQLAlchemy
+from ..db import db
+
+
+class MentionUser(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(50), unique=True, nullable=False)
+    company_name = db.Column(db.String(50), nullable=False)
```

Comment:
May have multiple company_names

Reply:
I originally wasn't going to have this here, I guess I just got confused when I saw company name as part of the sign up, guess I could still have a separate table for it, and just have it put one entry in when the user signs up.

Reply:
Yeah that makes more sense